data/presets/barrier/log.json

```diff
@@ -10,6 +10,5 @@
     "tags": {
         "barrier": "log"
     },
-    "name": "Fallen Tree",
-    "searchable": false
+    "name": "Barrier (Fallen Tree)"
```

Comment:
I'm only not quite 100% happy with this name still. I've asked on slack's #language channel for suggestions about how to name it better.